api/src/pcapi/local_providers/cinema_providers/cgr/cgr_stocks.py

```diff
@@ -7,15 +7,15 @@
 from pcapi.core.categories import subcategories_v2 as subcategories
 from pcapi.core.external_bookings.cgr.client import CGRClientAPI
 import pcapi.core.offerers.models as offerers_models
+from pcapi.core.offers import api as offers_api
 import pcapi.core.offers.models as offers_models
 import pcapi.core.offers.repository as offers_repository
 import pcapi.core.providers.models as providers_models
 from pcapi.local_providers.cinema_providers.constants import ShowtimeFeatures
 from pcapi.local_providers.local_provider import LocalProvider
 from pcapi.local_providers.providable_info import ProvidableInfo
 from pcapi.models import Model
-from pcapi.utils.date import get_department_timezone
-from pcapi.utils.date import local_datetime_to_default_timezone
+from pcapi.utils import date as utils_date
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do not leave false praise (which can actually be damaging). Do look for something to sincerely praise.  -->
**praise:** ‏👍🏼